priv/supabase/ingest_samples/gotrue.logs.prod.json

```diff
@@ -8,7 +8,44 @@
     "msg": "request started",
     "path": "/user",
     "remote_addr": "152.26.89.239",
-    "timestamp": "2023-03-23T14:36:26Z"
+    "timestamp": "2023-03-23T14:36:26Z",
+    "action": "some action",
+    "args": ["arg"],
+    "auth_event": {
+      "action": "action",
+      "actor_id": "123-123",
+      "actor_name": "some name",
+      "actor_username": "some username",
+      "actor_via_sso": false,
+      "log_type": "some type",
+      "traits": {
+        "channel": "some channel",
+        "provider": "google",
+        "user_email": "some@email.com",
+        "user_id": "123-123",
+        "user_phone": "123-123"
+      }
+    },
+    "client_id": "123-123",
+    "duration": "123sec",
+    "error": "some error",
+    "factor_id": "123-123",
+    "header": "some header",
+    "instance_id": "123-123-123",
+    "issuer": "some issuer",
+    "project": "some123ref",
+    "metering": false,
+    "message": "some message",
+    "login_method": "some method",
+    "provider": "google",
+    "referer": "example.com",
+    "remote_addr": "123.123.123",
+    "request_id": "123-123",
+    "source_type": "some type",
+    "status": "some status",
```

Comment:
```suggestion
    "status": 200,
```


Based on what I see on logflare prod, the status field is of integer type.

Reply:
![Screenshot 2023-07-05 at 12 14 59 PM](https://github.com/Logflare/logflare/assets/1639722/fcdac943-8ac2-4e21-9580-500f934f6178)

To clarify, the json field logged by gotrue is an integer, but vector transforms it to a string. May be ok to ignore if the change is too complicated.

Reply:
the seed payload should match the schema's types, since it skips vector transformations.  otherwise we'd have an integer column instead of string column being created.

Reply:
Yea ideally gotrue log should also match the schema type and it currently doesn't. Small thing though